src/dmd/identifier.d

```diff
@@ -138,6 +138,22 @@ nothrow:
         return idPool(buf.peekSlice());
     }
 
+    /***********************************************************
+     * Generate deterministic named Id based on the `loc` such that
+     * the naming is consistent across multiple compilations.
+     * `prefix` must be a null-terminated string.
+     */
```

Comment:
New functions should have standard Ddoc sections like `Returns:`  and `Params`.

Reply:
fixed